docs/source/api/v4/servers.rst

```diff
@@ -69,6 +69,11 @@ Request Structure
 	|                |          | the first page is 1. If ``offset`` was defined, this query parameter has no effect. ``limit`` must be defined to  |
 	|                |          | make use of ``page``.                                                                                             |
 	+----------------+----------+-------------------------------------------------------------------------------------------------------------------+
+	| asn            | no       | Return only the servers that have a cachegroup matching the provided ASN.                                         |
+	|                |          |                                                                                                                   |
+	|                |          |.. versionadded:: 4.1                                                                                              |
```

Comment:
I was expecting this to result in a malformed table, but the GHA would show a warning somewhere if that was true, so I was thrown for a bit of a loop.

I'm not sure why, but the table isn't malformed. Instead, this is causing the above text ("Return only the servers that have a cachegroup matching the provided ASN.") to be rendered as a block quote.

The `.. versionadded:: 4.1` directive needs a space between it and the `|`. Again, idk why that's causing this specific issue, but I confirmed that adding the space does indeed fix it.